### ISSUE TITLE
shx can't display remote image files

### ISSUE THREAD

**Issue by ricardog** [riscy/shx-for-emacs#28]
Thank you for developing **shx**. It really makes emacs shell buffers better.

When I open a remote shell, I can't use the `:view` command to display an image. I get the following error

```
ubuntu@vdc:~$ :view Snowflake.png 
convert: unable to open image `/ssh:ubuntu@vdc:/home/ubuntu/Snowflake.png': No such file or directory @ error/blob.c/OpenBlob/2924.
convert: no images defined `/tmp/tmp4vDiai.png' @ error/convert.c/ConvertImageCommand/3229.

```
Because `shx` is trying to run `convert` locally but passing it a tramp file name.

**Comment by riscy** [comment]
Yeah, this is definitely something that can be improved.  I've got my eyes on it. :)

**Comment by riscy** [comment]
This one is taking a little longer to figure out.  I'm not sure why, but running `(insert-image (create-image "xyz.png"))` in tramp/remote buffer seems to insert an empty square.  Meanwhile <kbd>C-x C-f xyz.png</kbd> or `:e xyz.png` opens a buffer with the image correctly displayed.

**Comment by emacsbliss** [comment]
wonder if we're able to make progress on this? really need this feature in my workflow, thanks!

**Comment by riscy** [comment]
Thank you for giving this a bump.  I'll prioritize seeing if I can find a way around the the hurdle I encountered in the last attempt.

**Comment by emacsbliss** [comment]
another bump see where we are at with this..
